.travis.yml

```diff
@@ -310,8 +310,8 @@ matrix:
     # Add this env var so we can assert in the tests that the library is installed correctly
     - export TEST_R_WITH_ARROW=TRUE
     - pushd ${TRAVIS_BUILD_DIR}/r
-    after_success:
-    - Rscript ../ci/travis_upload_r_coverage.R
+    #after_success:
+    #- Rscript ../ci/travis_upload_r_coverage.R
```

Comment:
@nealrichardson Is there something more to do to disable R coverage computation?